src/components/Api.js

```diff
@@ -0,0 +1,212 @@
+import Section from './Section.js';
+import {createNewCard} from '../utils/utils.js';
+import {cardContainerClass} from '../utils/constants.js';
+import { userInfo, formValidatorCardModal } from '../pages/index.js';
+
+export default class Api{
+    constructor(){
+      this._baseUrl = 'https://around-api.en.tripleten-services.com/v1',
+      this._headers = {
+        authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
+        "Content-Type": "application/json"
+      };
+      this._headers1 = {
+        authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
+      }
+    }
+
+    loadUserInfo(){
+      return fetch(`${this._baseUrl}/users/me`, {
+        headers: this._headers1
+      })
+      .then(res => {
+        if(res.ok){
+          return res.json();
+        } else{
+          return Promise.reject(`Error: ${res.status}`);
+        }
+      })
+      .then(data => {
+        userInfo.setUserInfo(data);
+        document.querySelector('.profile__image').src = data.avatar;
+      })
+      .catch(err => {
+        console.error(err);
+      })
+    }
+
+    loadCards(){
+      return fetch(`${this._baseUrl}/cards`, {
+        headers: this._headers1
+      })
+      .then(res => {
+        if(res.ok){
+          return res.json();
+        } else{
+          return Promise.reject(`Error: ${res.status}`);
+        }
+      })
+      .then(data => {
+        const renderCards = new Section({items: data, renderer: (cardItem) => {
+          const cardElement = createNewCard(cardItem);
+
+          if(cardItem.isLiked){
+            cardElement.querySelector('.card__love-icon').classList.add('card__love-icon_background_black');
+          } else{
+            cardElement.querySelector('.card__love-icon').classList.remove('card__love-icon_background_black');
```

Comment:
Now only class `Card` knows every card selector, so you need to encapsulate the likes logics inside its methods and call them in `index.js`